### ISSUE TITLE
Dialog box problem - could be my problem only

### ISSUE THREAD

**Issue by tiera3** [magefree/mage#12539]
Often (feels like about half the time, but I haven't taken notes to determine frequency) I an unable to type more than one character is a dialog box. (The character typed is highlighted, and each time I type another character it overwrites the previous one and highlights the new one.)

Workaround - I type the needed text in notepad and copy, then paste into the dialog box - this works fine.

I know that intermittent problems are the hardest to diagnose, but I just thought I would mention it in case:
a) Someone knows of the cause and/or a simple solution, or
b) Others are having the same issue.

**Comment by ssk97** [comment]
I've occasionally had that problem, but haven't found a consistent repro for it and I suspect it's a library bug that xmage can't easily fix (aside from by finally updating everything).

Simply clicking outside the box and then clicking back in the text box has fixed it every time it's happened for me.

**Comment by JayDi85** [comment]
Yes, I know about that, but don’t know the reason. Something wrong with text box editor and auto-selection (caret position).
